app/src/main/java/org/mozilla/fenix/trackingprotection/TrackingProtectionOverlay.kt

```diff
@@ -96,6 +96,7 @@ class TrackingProtectionOverlay(
         trackingOnboardingDialog.apply {
             setContentView(layout)
             setCancelable(false)
+            setTitle(" ")
```

Comment:
Have you considered `requestWindowFeature(Window.FEATURE_NO_TITLE)` ?
Would look less like a hack.

Reply:
Yes, tried it, does not work.

Reply:
Well, this is quite unfortunate but at least we have a fix.
Thanks for looking into it.